docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1854,8 +1875,10 @@ async function getStatsForVideoPane(meeting, videoPane) {
 
   // there is no public API in WCME to get the transceiver of a receive slot, but
   // this is javascript so we can access private fields to get it anyway until we have some API for this
-  // todo: 'VIDEO-MAIN' will need to change when we support screen sharing (SPARK-377812)
-  const transceiver = multistreamConnection.recvTransceivers.get('VIDEO-MAIN')?.find((t) => t.receiveSlot === wcmeReceiveSlot);
+  const transceiver = [
+    ...(multistreamConnection.recvTransceivers.get('VIDEO-MAIN') || []),
```

Comment:
I think it should be fine just for the stats , But are we adding some stats api in the WCME code itself

Reply:
yeah Tyler recently added `getTransceiverStats()` API to WCME here: https://sqbu-github.cisco.com/CPaaS/web-client-media-engine/pull/124/ but I think that is still not good enough as it doesn't let me easily associate which transceiver is for which receive slot, so we may need to improve that `getTransceiverStats()` API in the future